homeassistant/components/mochad.py

```diff
@@ -33,18 +37,33 @@
     })
 }, extra=vol.ALLOW_EXTRA)
 
+CONFIG_SCHEMA_MQTT = vol.Schema({
```

Comment:
This isn't used.

Reply:
Move this schema to be a key under the main component schema.

Reply:
Removed the schema. Also the addition is not breaking. If no mqtt brokes specified, mochad will work as before for sending commands to switches. No receiving events from sensors. The latter requires mqtt.